package.json

```diff
@@ -6,8 +6,8 @@
   "author": "Marcello Milhomem Albuquerque <marcello.m.albuquerque@gmail.com>",
   "license": "MIT",
   "scripts": {
-    "test": "mocha",
-    "test:watch": "mocha --watch",
```

Comment:
do not use this, it does not work on windows. If necessary install `cross-env` as dev dependency and use `cross-env ENV=test mocha --watch` etc instead